ParcialProgramacion4/src/Convocatoria.java

```diff
@@ -12,13 +12,54 @@ public class Convocatoria {
     private Hashtable<Habilidad,Integer>requisitos;
     private ArrayList<Empleado>asignados;
 
-    public Convocatoria(int codigo, Puesto puesto, Fecha fecha, int cantEmpleadosRequeridos) {
+    public Convocatoria(int codigo, Puesto puesto, Fecha fecha, int cantEmpleadosRequeridos, Hashtable<Habilidad,Integer> requisitos) {
         this.codigo = codigo;
         this.postulados = new ArrayList<Empleado>();
```

Comment:
En estos casos, esta permitido hacer este tipo de escritura
```suggestion
    public Convocatoria(
        int codigo,
        Puesto puesto,
        Fecha fecha,
        int cantEmpleadosRequeridos,
        Hashtable<Habilidad, Integer> requisitos
    ) {
        this.codigo = codigo;
        this.postulados = new ArrayList<Empleado>();
```

Reply:
si pero no se si lo va a querer el profe
?

Reply:
Son dos formas de escribir lo mismo